### PR TITLE
[Blog] Translation dubbo-annotation-driven.md

### PR DESCRIPTION
**Pull request by complone**
## What is the purpose of the change

XXXXX

## Brief changelog

blog:
- en-us
  - dubbo-annotation-driven.md 

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/dubbo-website/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Test your code locally by running `docsite start`, and make sure it works as expected.
- [ ] Make sure no files under build directory is added.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

finished dubbo-annotation-driven blog translation

### CODE REVIEW

File: blog/en-us/dubbo-annotation-driven.md [hunk 1]
```diff
@@ -0,0 +1,806 @@
+#Annotation-Driven
+
+
+
+
+##Annotation-Driven
```
Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: markdown format require a space between title and '#'

File: blog/en-us/dubbo-annotation-driven.md [hunk 2]
```diff
@@ -0,0 +1,806 @@
...
+
+
+
+
+
+##### 1. Insufficient support for annotations
+
+
+
+In Dubbo 2.5.7version before, Dubbo provides two core notes `@Service`and `@Reference`, Dubbo Dubbo and service delivery service references are used.
```
Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: maybe use 'version 2.5.7'?

Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: Dubbo Dubbo ?

File: blog/en-us/dubbo-annotation-driven.md [hunk 3]
```diff
@@ -0,0 +1,806 @@
...
+
+
+Among them, `@Service` as an XML element <dubbo:service>replacement annotated with Spring Framework @org.springframework.stereotype.Serviceis similar to the service provider Dubbo service exposure. Corresponding to `@Reference` it is an alternative <dubbo:referenceelement, similar to Spring `@Autowired`.
```
Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: space needed between <dubbo:service> and replacement

File: blog/en-us/dubbo-annotation-driven.md [hunk 4]
```diff
@@ -0,0 +1,806 @@
...
+> >In the project, I used it.
+> >
+> >```java
+> >@Service
+> >@Transactional
+> >@com.alibaba.dubbo.config.annotation.Service
+> >public class SUserJpushServiceImp
+> >```
+> >
+> >In the form of to expose the service. But when publishing services, interface class is by
```
Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: Please compare with chinese version to check this section.

File: blog/en-us/dubbo-annotation-driven.md [hunk 5]
```diff
@@ -0,0 +1,806 @@
...
+> >``
+> >obtaining a form, just, I have used the service @Transactional annotation, the object is a proxy. So the interface obtained is Spring's proxy interface...
+
+
+
+Many enthusiastic partners not only discovered this legacy, but also proposed some repairs. At the same time, in order to better adapt to the Spring life cycle and transition Dubbo completely to the annotation-driven programming model, a new Dubbo component scan annotation was introduced `@DubboComponentScan`.
+
+
+
+> Note: `<dubbo:annotation>` the Spring AOP issue will be `2.5.9`fixed in: [https://github.com/alibaba/dubbo/issues/1125](https://github.com/alibaba/dubbo/issues/1125)
```
Comment (htynkn) [inline blog/en-us/dubbo-annotation-driven.md]: space needed after `2.5.9` and also this section need change, should be fixed in XX

File: blog/en-us/dubbo-annotation-driven.md (hunk 4 above)
Comment (complone) [reply blog/en-us/dubbo-annotation-driven.md]: Ok, I will see it.

Comment (complone) [reply blog/en-us/dubbo-annotation-driven.md]: Hello, I have corrected

**Comment by htynkn** [comment]
Could you also check whole page? there are many order issue in your english version compared with chinese version

**Comment by lexburner** [comment]
The pr seems a bit confused, could you repair it and send a reopen pr? 
Reopen if need.

**Pull request closed without merging**